src/app/api/og/photographer/route.ts

```
@@ -0,0 +1,55 @@
+import { ImageResponse } from "workers-og";
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)